src/inverted-index.js

```diff
@@ -104,6 +107,9 @@ class InvertedIndex {
   }
```

Comment:
### Critical

I already gave you feedback to refrain from using `for loops` for Arrays, you should use the `forEach` API instead. You corrected it only in the instance I pointed out, you need to pay more attention to detail to go above and beyond to correct all instances of the issue(s).